android/build/common.xml

```diff
@@ -355,8 +355,8 @@ Common ant tasks and macros for building Android-based Titanium modules and proj
 	<macrodef name="generate.build.properties">
 		<attribute name="dir" default="${dist.classes.dir}/titanium"/>
 		<sequential>
-			<mkdir dir="@{dir}/org/appcelerator/titanium"/>
-			<propertyfile file="@{dir}/org/appcelerator/titanium/build.properties"
+			<mkdir dir="@{dir}/assets"/>
+			<propertyfile file="@{dir}/assets/build.properties"
```

Comment:
To avoid collision, maybe we should rename the file to:
`assets/ti.build.properties`

What do you think?